dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ConversionExceptionResource.nlsprops

```diff
@@ -39,7 +39,7 @@
 
 3006=[{0}] must be of even length to be converted to a byte array.
 
-3007=The object [{0}] object of the [{1}] class cannot be converted to [{2}].  Ensure that the [{2}] class is on the CLASSPATH.  You might need to use alternate API passing in the appropriate class loader as required, or setting it on the default ConversionManager.
+3007=The object [{0}], of class [{1}], could not be converted to [{2}].  Ensure that the class [{2}] is on the CLASSPATH.  You may need to use alternate API passing in the appropriate class loader as required, or setting it on the default ConversionManager
```

Comment:
Change back to `The [{0}] object of the [{1}] class cannot be converted to [{2}]. Ensure that the [{2}] class is on the CLASSPATH. You might need to use alternate API passing in the appropriate class loader as required, or setting it on the default ConversionManager.`